### PR TITLE
add pytorch 1.11 tests

### PR DESCRIPTION
**Pull request by miaoshasha**
pytorch:
add 1.11 tests, remove 1.9 tests

To be merged after RCs are created.

### CODE REVIEW

File: tests/pytorch/r1.11/python-ops.libsonnet [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
+// Copyright 2020 Google LLC
```
Comment (JackCaoG) [inline tests/pytorch/r1.11/python-ops.libsonnet:1]: I am guessing we should change it to 2022?

File: tests/pytorch/r1.9/colab.libsonnet [hunk 2]
```diff
@@ -1,55 +0,0 @@
-local common = import 'common.libsonnet';
```
Comment (will-cromar) [inline tests/pytorch/r1.9/colab.libsonnet:1]: Can you include the colab notebook tests in r1.11?

File: tests/pytorch/r1.11/python-ops.libsonnet (hunk 1 above)
Comment (miaoshasha) [reply tests/pytorch/r1.11/python-ops.libsonnet]: done

File: tests/pytorch/r1.9/colab.libsonnet (hunk 2 above)
Comment (miaoshasha) [reply tests/pytorch/r1.9/colab.libsonnet]: added

File: tests/pytorch/r1.11/python-ops.libsonnet (hunk 1 above)
Comment (JackCaoG) [reply tests/pytorch/r1.11/python-ops.libsonnet]: All of the files in this pr seems to share this typo, can you fix them?

Comment (miaoshasha) [reply tests/pytorch/r1.11/python-ops.libsonnet]: done

**Review by JackCaoG — APPROVED** [review]
I will approve this pr to kick of test for now. If there is anything we need to change we can do that in the following pr.

**Pull request merged**